packit_service/worker/helpers/testing_farm.py

```diff
@@ -1408,21 +1410,21 @@ def prepare_and_send_tf_request(
         )
 
     @implements_fedora_ci_test("installability")
-    def _payload_installability(self, compose: str) -> dict:
+    def _payload_installability(self, distro: str, compose: str) -> dict:
         git_repo = "https://github.com/fedora-ci/installability-pipeline.git"
         git_ref = (
             commands.run_command(["git", "ls-remote", git_repo, "HEAD"], output=True)
             .stdout.strip()
             .split()[0]
         )
 
-        if self.koji_build.target == "rawhide":
+        if distro == "fedora-rawhide":
```

Comment:
Any specific reason why you switched? :eyes:

Reply:
It makes no difference here, but it does below (lowercased compose might not always be correct, distro is a better bet), so the change here is just for consistency sake.